### DISCUSSION TITLE
Integrate Windows NT authentication for unattended access

### DISCUSSION THREAD

**Discussion by florianwgnr** [rustdesk/rustdesk#8020] [General]
Current situation: 

- A permanent password allows for unattended remote connections. 
- The permanent password is set during setup or via command-line parameters. 
- Connection options besides permanent password:
  - one time password
  - allow by click

Feature request: 
Provide another connection option called "Windows NT authentication", which uses the built-in authentication method from Windows to use existing user accounts, e.g. the Administrator (or other users?) to check the provided password against. 

Useful situations: 
- Fully replace Remote Desktop by using the same authentication "process"
- IT and OT environments with built-in user accounts or domain accounts
  - especially domain accounts can be disabled centrally in the ActiveDirectory and therefore no login is possible anymore, e.g. when somebody leaves the company
- avoid saving additional passwords (e.g. the permanent password), as these can easily be overlooked when making or forcing changes to the windows credentials

**Comment by jackadam1981** [comment]
If you want to integrate Windows domain authentication like this, why not use Windows Remote Desktop?

**Comment by florianwgnr** [comment]
> If you want to integrate Windows domain authentication like this, why not use Windows Remote Desktop?

I work in an engineering company where machines with with windows OS are built. You can't use RDP on these machines because it locks the screen of the machine / HMI and an operator in front of the machine is no longer able to see or control it. That is why I am asking for such functionality in RustDesk.

**Comment by jackadam1981** [comment]
RUSTDESK SERVER PRO
RustDesk Server Pro has more features compared to the open source version.

No concurrent connection limit, aka no limit of simultaneous connections (OSS version has no limit either, but TeamViewer etc. have this limit)
[Web console](https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/console/)
API
[OIDC](https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/oidc/), [LDAP](https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/ldap/), [2FA](https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/2fa/)
Address book
Rename
Log management (Connection, file transfer, alarm, etc.)
Device management
[Security Settings sync](https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/strategy/)
[Access control](https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/permissions/)
[Multiple relay servers](https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/relay/) (automatically selects your closest relay)
[Custom client generator](https://rustdesk.com/docs/en/self-host/client-configuration/#1-custom-client-generator-pro-only)

**Comment by Semitour** [comment]
Hi,

We seems to have the same needs here, like in [this feature request ](https://github.com/rustdesk/rustdesk-server-pro/issues/270)

What we are looking for in an enterprise situation, is to have full remote access on 100+ machines, without the need to manage passwords for all the clients. The method @florianwgnr is talking about helps to authenticate and allow accès from a central source of truth wich is Windows active directory in this case

This is the only feature missing before we buy subscription. I guess we are not the only ones.
Please consider implementing this feature so we can leave those horrible solution like Teamviewer and LogMeIn. Be that hero!! 😍
